games/xworld3d/x3item.h

```diff
@@ -200,7 +202,9 @@ namespace std {
 template <>
 struct hash<simulator::Vec3> {
     size_t operator()(const simulator::Vec3& l) const {
```

Comment:
This hash function is no longer used. So remove it.

Reply:
there is someplace that still uses it

Reply:
Where is it used? I think indexing the loc should only be done in Python.